lib/Dialect/FIRRTL/Transforms/LowerSignatures.cpp

```diff
@@ -199,13 +209,19 @@ computeLoweringImpl(FModuleLike mod, PortConversion &newPorts, Convention conv,
                 annosForFieldIDRange(ctx, annos, fieldID, lastId)},
                portID,
                newPorts.size(),
-               fieldID});
+               fieldID,
+               hasInternalPaths ? InternalPathAttr::get(ctx)
+                                : InternalPathAttr{}});
         } else {
           for (size_t i = 0, e = vector.getNumElements(); i < e; ++i) {
             if (failed(computeLoweringImpl(
                     mod, newPorts, conv, portID, port, isFlip,
                     name + "_" + Twine(i), vector.getElementType(),
-                    fieldID + vector.getFieldID(i), syms, annos)))
+                    fieldID + vector.getFieldID(i), syms, annos,
+                    hasInternalPaths,
+                    internalPath
+                        ? std::optional{*internalPath + "[" + Twine(i) + "]"}
```

Comment:
ExtModules should be the only things with internalPaths and they should never have probes inside a bundle or vector.

I'm not sure what these cases are trying to handle.